mlcluster-vpc.template

```diff
@@ -101,7 +107,16 @@ Parameters:
     AllowedValues:
       - standard
       - gp2
+      - gp3
     Default: gp2
+  VolumeIops:
+    Description: The number of I/O operations per second for a gp3 volume
+    Type: String
```

Comment:
Did we find any validation check to change the type and allow values from 3000 to 16000 ?

Reply:
added constraints